packages/app/components/Queue/Student/StudentQueue.tsx

```diff
@@ -267,7 +267,7 @@ export default function StudentQueue({ qid }: StudentQueueProps): ReactElement {
   );
 
   if (queue && questions) {
-    if (!queue.isOpen) {
+    if (!queue.isOpen || queue.isDisabled) {
```

Comment:
if we updated the checkIsOpen function to set isOpen based on being disabled as well, do we need to also check queue.isDisabled?